flexmeasures/api/v3_0/tests/test_sensors_api.py

```diff
@@ -40,18 +47,47 @@ def test_fetch_sensors(
     if use_pagination:
         query["page"] = 1
 
+    if search_by:
```

Comment:
I don't understand what these two lines are meant to do. `search_by` clearly should not be used as a flag.

With [my comment](https://github.com/FlexMeasures/flexmeasures/pull/1191#discussion_r1781217158) I meant to suggest that with `search_by` we can either test looking up sensors by a unit or by a search word filter.

Reply:
oh, so you mean like

search_by = m/s

or

search_by = "temp sensor"

Reply:
No, I meant it like my example, is that example unclear?

Reply:
You like the below

search_by = unit
search_value = m/s

and in another case

search_by = filter
search_value = "temp sensor name"

Reply:
yes

Reply:
oh ok, sorry about the confusion, i will update that.